BuildRelease.bat

```diff
@@ -1,2 +1,3 @@
+del .\release\hardCrafting2_0.17.2.zip
 java -jar BuildRelease1.2.jar
-pause
\ No newline at end of file
+copy .\release\hardCrafting2_0.17.2.zip d:\Data\Games\factorio\develop\mods
```

Comment:
I removed these changes to BuildRelease.bat on the official repository because they seem specific to your setup. You can add another bat file and gitignore it if you need these commands ;)